glue/core/data_factories/fits.py

```diff
@@ -176,7 +176,7 @@ def casalike_cube(filename, **kwargs):
     from astropy.io import fits
 
     result = Data()
-    with fits.open(filename, **kwargs) as hdulist:
+    with fits.open(filename, ignore_missing_end=True, **kwargs) as hdulist:
```

Comment:
Just an observation -- If `ignore_missing_end` is also provided via `kwargs`, there is going to be conflict.